docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,92 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
+
+### The Scalable Pixel Streaming Frontend TypesScript example
+The TypeScript example is a simple HTML, CSS and TypeScript implementation of what a user could create to initialize the Scalable Pixel Streaming Frontend library. Its role is to instantiate the library's components and help start the Scalable Pixel Streaming connection to the signalling server.
+
+## Getting Started; installation and consumption
+### Building for Development vs Production
+The Scalable Pixel Streaming Frontend packages contain several npm scripts that can be used to build the library and example for either development or production. When building for development this will enable source maps for debugging. When building for production this will disable source maps, reduce console output and minify the distributed JavaScript files. Below are a list of npm scripts for both the library and example and what each command does:
+
+#### library scripts
+Please ensure that all library scripts are executed from the `library` directory and a user must first run `npm install` to install all the Libraries dependencies.  
+- `npm run build-dev`: Build the library in development mode
+- `npm run build-prod`: Build the library in production mode
+
+#### example scripts
+Please ensure that all example scripts are executed from the `examples/typescript` directory. It is always recommended that a user installs the library first as the example requires the library. Contrary to this however, `build-all-dev` and `build-all-prod` do not require the library to be installed and built first and will install and build both the library and example and all dependencies.
+- `npm run build-dev`: Build the library in development mode
+- `npm run build-prod`: Build the library in production mode
+- `npm run serve-dev`: Serve the example locally using the library in development mode
+- `npm run serve-prod`: Serve the example locally using the library in production mode
+- `npm run symlink`: Links the library to the example for consumption
+- `npm run build-all-dev`: Build the library and the example in development mode and link the library to the example for consumption
+- `npm run build-all-prod`: Build the library and the example in production mode and link the library to the example for consumption
+
+### Installing the Scalable Pixel Streaming Frontend library from Github
```

Comment:
Capitalise Library here when used with Scalable Pixel Streaming. Fix: GitHub

Reply:
Resolved in 4820831c542f48909f428f142606b7c814b6ede6